File: app/src/main/java/com/github/gfx/helium/util/UserAgent.java
package com.github.gfx.helium.util;

import com.github.gfx.helium.BuildConfig;

import android.os.Build;

public class UserAgent {
    public static String build() {
        return BuildConfig.APPLICATION_ID + "/" + BuildConfig.VERSION_NAME
                +  " Android/" + Build.VERSION.RELEASE
                + " " + Build.DEVICE
                + " " + Build.BRAND
                + " " + Build.MODEL;

    }
}

File: app/src/main/java/com/github/gfx/helium/util/UserAgent.kt
package com.github.gfx.helium.util

import android.os.Build
import com.github.gfx.helium.BuildConfig

object UserAgent {

    fun build(): String {
        return BuildConfig.APPLICATION_ID + "/" + BuildConfig.VERSION_NAME +
                "; Android/" + Build.VERSION.RELEASE + "; " +
                Build.DEVICE + "; " +
                Build.BRAND + "; " +
                Build.MODEL
    }
}
